### DISCUSSION TITLE
Adókód változásra speifikció kellene

### DISCUSSION THREAD

**Discussion by Macskafarka** [nav-gov-hu/eVAT#91] [Kérdések és válaszok - Gépi interfész] [answered]
A specifikációnak arra is ki kellene térnie, hogy kinek mi a teendője, ha egy már létező adókód tartalma teljesen megváltozik. Erre az esetre mindenképpen kell valami leírás, hogy kinek mi a teendője, hol és hogyan kell közzétennie, mi történjen a már leadott bevallásokkal (önrevíziózni kell-e, vagy a NAV átforgatja, és újra le kell kérni, és tárolni a módosított bevallást az ERP rendszerben stb) , mi történjen az éppen folyamatban lévő bevallással. Indokolás: alig telt el az indulás óta 14 nap, és máris volt példa a kódok tartalmának megváltozására, gondolok itt a VM és VV végű kódokra. Úgy gondolom, hogy talán ilyenkor le kellene zárni a hibás kódok érvényességét, és új kódot nyitni, de erre nincs felkészítve a kódrendszer sem.

**Comment by renced42** [comment, marked as answer]
Kedves @Macskafarka 

Köszi az észrevételt, készítünk egy ilyen leírást.
